docs/Manpage.md

```diff
@@ -1647,6 +1647,8 @@ to send a notification when the autoupdate process has finished successfully.
 
 * `--upgrade`:
   Automatically upgrade your installed formulae. If the Caskroom exists locally Casks will be upgraded as well. Must be passed with `start`.
+* `--greedy`:
+  Upgrade casks with --greedy. See brew(1).Must be passed with `start`.
```

Comment:
```suggestion
  Upgrade casks with --greedy. See brew(1). Must be passed with `start`.
```

I'll investigate tomorrow what does this generation and how to make it have a space after the `.` there. Otherwise it'll just regenerate badly the next time.

Reply:
Ah, it was https://github.com/Homebrew/homebrew-autoupdate/pull/85.